### PR TITLE
Update release notes for 1.82

### PR DESCRIPTION
**Pull request by vissarion**

### CODE REVIEW

**Review by barendgehrels — APPROVED** [review]
Thanks!

There were also some fixes in union (but no problem if it is not added)

File: doc/release_notes.qbk [hunk 1]
```diff
@@ -27,9 +27,15 @@
 
 * [@https://github.com/boostorg/geometry/pull/1045 1045] Support geographic buffer for (multi)linestrings and (multi)polygons
 
+[*Improvements]
+
+* [@https://github.com/boostorg/geometry/pull/1114 1114] Comparative distance: Add support for dynamic geometries
```
Comment (awulkiew) [inline doc/release_notes.qbk:32]: Is it a problem that the name of the algorithm is comparable distance, not comparative?

**Review by awulkiew — APPROVED** [review]

**Comment by awulkiew** [comment]
Thanks!

**Comment by vissarion** [comment]
Thanks! Updated also for boost website https://github.com/boostorg/website/pull/745

**Pull request merged**